docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/enforcement.adoc

```diff
@@ -140,7 +140,7 @@ image::application-security/enforcement2.0.gif[]
 [#add-label]
 === Add Labels
 
-You can apply labels to both default and custom policies when you create or edit an existing policy. Labeling enforcement rules provides a filtering mechanism. When you apply a label to enforcement rules, those rules will only apply to repositories with the corresponding label. Labels can be reused and assigned across multiple enforcement rules, allowing for the grouping of policies and the creation of enforcement rules for these labeled groups.
+You can add labels to both default and custom policies. After applying labels to policies, you can go to Enforcement and add labels. When PR scans run, only the policies that have the same labels as those in the enforcement rule and meet the severity thresholds will be applied. Policies that do not match these criteria will not trigger PR comments or cause a hard failure in the PR. The Projects Enforcement Level filter can help you focus only on these relevant findings. If you remove the filters, all findings will be displayed, regardless of whether they meet the enforcement rule criteria.
```

Comment:
You can add labels to both default and custom policies. After applying labels, you can create enforcement rules that target specific policy labels and severity thresholds. When PR scans run, only policies matching these criteria will trigger PR comments or cause PR failures. The Projects Enforcement Level filter allows you to focus on relevant findings. Removing these filters will display all findings, regardless of enforcement rule criteria.

Reply:
LGTM. I updated the change